.changeset/red-pens-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': patch
```

Comment:
```suggestion
'@backstage/plugin-scaffolder-backend': minor
```

Sorry I forgot to point out that new features to 1.x packages should be a minor bump. I tried to be helpful and push an update to your branch but it looks like that's did not get reflected in this PR.

Reply:
From the description minor sounded like a breaking change and my change is not ^^